### DISCUSSION TITLE
VM Proxmox mit HMIP-HAP als Funkmodul? Warum brauche ich ein HM-RFUSB ?

### DISCUSSION THREAD

**Discussion by PVMike** [OpenCCU/OpenCCU#2873] [🇩🇪 Fragen&Antworten] [answered]
### Describe the solution you'd like

Ich nutze seit längerem Proxmox mit diversen Funktionalitäten, und habe nun RaspberryMatic installiert.
Mein Server steht in einem "sicheren", leider auch "Funkdichten" Kellerraum.
Ich wollte mit einer über LAN verbundenen HMIP-HAP das Funknetzwerk im Haus ausbauen, und da ich zwei davon habe beide als RF-Gateways einsetzen um meine HM-IP Geräte anzusteuern (Heizungm, Fenster etc,) 
Beim Anlernen der HMIP-HAP in RaspMatic kommt eine Fehlermeldung, dass kein Funkmodul erkannt wurde.

Das sollte man ändern, soweit technisch möglich, dass RaspMatic auch ohne USB-Funkmodul funktioniert, und nur mit einem oder mehrerern HMIP-HAP Geräten betrieben werden kann.

### Describe alternatives you've considered

keine Alternative, ich habe das Funkmodul nun bestellt, aber es ist ärgerlich, weil es da wo es am Server angeschlossen wird nichts empfangen wird können... ich hoffe das die HABs dann wenigstens angelernt werden können, wenn das USB Teil im Server steckt

### Is your feature request related to a problem?

ja, ohne RFUSB läuft mein Setup nicht.

### Additional information

Gerne auch Rückmeldung wenn ich etwas falsch verstanden oder sonst wie falsch gemacht haben sollte.
Meine HABs wurden auf Werkeinstellungen zurückgesetzt bzw waren out-of-the box

**Comment by jens-maus** [comment, marked as answer]
> Gerne auch Rückmeldung wenn ich etwas falsch verstanden oder sonst wie falsch gemacht haben sollte. Meine HABs wurden auf Werkeinstellungen zurückgesetzt bzw waren out-of-the box

Hier wurde in der Tat etwas falsch verstanden. Das gemeinhin genannte "Funkmodul" wie HmIP-RFUSB oder RPI-RF-MOD oder HM-MOD-RPI-PCB sind mehr als nur reine Funkmodule. Sie bringen gewisse Verschlüsselungsroutinen mit sich die für den homematicIP Betrieb unablässlich sind. Daher erfüllen Sie mehr den Charakter eines Koprozessor. Ergo ist es absolut notwendig ein HmIP-RFUSB oder ähnliches "Funkmodul" bzw. "Koprozessor" direkt mit RaspberryMatic oder einer CCU verbunden zu haben wenn man homematicIP machen will (egal ob Funk oder Wired). Das bedeutet dann aber auch das auch die HmIP-HAP nicht das problem lösen werden, auch hier braucht man trotzdem lokale das HmIP-RFUSB egal ob dieses die Geräte dann erreichen kann oder nicht. Teilweise kann es sogar notwendig sein die Antenne dieser Funkmodule gänzlich abzuschneiden damit dieser nur als reine Koprozessoren arbeiten und nicht selber funken und damit im Zweifelsfall auch Geräte aber mit schlechter Sende-/Empfangsqualität erreichen.

All diese Informationen sind jedoch recht gängiges KnowHow das man in diversen Foren finden kann und für die es kein Feature Request oder ähnliches hier braucht. Deshalb verschiebe ich das ganze (erneut) in den Diskussionsbereich – wo es hingehört.

**Comment by Baxxy13** [comment]
> keine Alternative, ich habe das Funkmodul nun bestellt, aber es ist ärgerlich, weil es da wo es am Server angeschlossen wird nichts empfangen wird können...

Das du um ein direkt angeschlossenes "Funkmodul" nicht herum kommst hat ja Jens schon gesagt.
Du kannst aber den HmIP-RFUSB einstecken und nutzen ohne das eigentliche Funkmodul aufzulöten.
Dann geht die gesamte IP-Kommunikation über deine HAP's und im funkdichten Keller kann erst gar nicht versucht werden zu funken.
